apps/acrs/sections/summary-data-sections.js

```diff
@@ -115,6 +115,37 @@ module.exports = {
       }
     ]
   },
+  'family-in-your-referral': {
+    steps: [
+      {
+        steps: '/parent',
+        field: 'parent',
+        parse: (list, req) => {
+          if (!req.sessionModel.get('steps').includes('/parent')) {
+            return null;
+          }
+          return req.sessionModel.get('parent') === 'yes' ?
+            'Yes' : 'No';
+        }
+      },
+      {
+        step: '/parent-summary',
+        field: 'referred-parents',
+        parse: (list, req) => {
+          if (req.sessionModel.get('referred-parents') === undefined ||
+            req.sessionModel.get('referred-parents').aggregatedValues.length < 1) {
```

Comment:
If `list` is already populated with the correct aggregation array should it be the subject of this condition?

Reply:
Have replaced this somewhat of a placeholder parsed value with a dump of all the fields. We will have to replace this when we have worked out a good method of displaying all the aggregate fields in the summary pages.